docs/team.rst

```diff
@@ -21,218 +21,290 @@ folks.
-the wider Write the Docs community. The stories in each issue are distilled from
-conversations that happen across the WTD Slack, which are curated and written by
-a team of volunteers.
+Infrastructure
+..............
 
 Responsibilities
-````````````````
+  - Maintain Website infrastructure and community tools
+  - Triage and resolve technical issues 
+  - Review and support incoming proposed changes to the Website (GitHub)
+  - Technical support for other teams (i.e. Slack, Newsletter, Conferences)
+  - Maintain and develop video archives and other online resources
+  - Maintain Website layout and navigation
+  - Maintain the WEP system
 
-* Curate Slack conversations and distill them into brief newsletter stories
-* Write a monthly update on general goings-on in the community
-* Round up any Write the Docs events or meetups for the upcoming month
-* Assemble and publish the monthly newsletter to the mailing list and `the website <https://www.writethedocs.org/blog/archive/tag/newsletter/>`_
-
-The full description of the :doc:`newsletter process </organizer-guide/newsletter/newsletter-process>` is available in our Organizer Guide.
+Membership
+  - Team members must have previous technical experience
 
 Members
-```````
+  * `Sasha Romijn <https://twitter.com/mxsash>`_ *
+  * `Samuel Wright <https://twitter.com/plaindocs>`_
+  * `Eric Holscher <https://twitter.com/ericholscher>`_
 
-* `Beth Aitman <https://twitter.com/baitman>`_ *
-* `Hillary Fraley <https://github.com/hillaryfraley>`_
-* `Jennifer Rondeau <https://twitter.com/bradamante>`_
-* `Claire Lundeby <https://twitter.com/clairelundeby>`_
-* Royce Cook
-* Volunteers welcome!
+WTD Enhancement Proposals (WEP)
+...............................
 
+Responsibilities
+  - Triage incoming WEPs according to the WEP process that is defined in `WEP0 <https://github.com/writethedocs/weps/blob/master/accepted/WEP0000.rst>`__.
+  - Maintain and develop WEP format, templates, and requirements.
 
-Support Team
-------------
+Membership
+  - Currently staffed by the CoreOps team with input on proposals from the Community Council
 
-Friendly folk on the other end of the email addresses who answer miscellaneous
-questions about all and sundry. Anything from requests for information, to
-billing and technical issues.
+Code of Conduct (CoC)
+.....................
 
-Roles
-`````
+Responsibilities
+  - Maintain and develop community-wide CoC policy
+  - Monitor and process requests from the global CoC inbox
+  - Consult and support CoC escalations from other teams
+  - Semi-annual publication of CoC transparency reports from the whole community
 
-* Respond to emails from the community
-* Help folks with anything they need :)
+Membership
+  - Team members must have some previous experience with CoC handling
 
 Members
-```````
+  - Each conference has its own CoC team
+  - Meetups and Slack moderation teams perform their own processing
```

Comment:
Seems we need a global COC team as well? Should we define that as the CoreOps team for now?

```suggestion
  - Meetups and Slack moderation teams perform their own processing
  - A global CoC team is staffed by CoreOps, and acts as a fallback for the other CoC functions within the community. 
```